runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java

```diff
@@ -297,6 +306,51 @@ private void playerSkillLookup(ChatMessageType type, SetMessage setMessage, Stri
 		}
 	}
 
+	/**
+	 * Calculates a simple equation and changes the original message to the
+	 * response.
+	 *
+	 * @param messageNode The chat message containing the command.
+	 * @param eqn         The equation to calculate.
+	 */
+	private void calculate(MessageNode messageNode, String eqn)
+	{
+		final ChatMessageBuilder builder = new ChatMessageBuilder();
+		
+		try
+		{
+			final Expression expr = new ExpressionBuilder(eqn).build();
+			final String result = String.valueOf(expr.evaluate());
+			builder.append(ChatColorType.NORMAL)
+				.append("Calc: ")
+				.append(eqn)
+				.append(" = ")
+				.append(ChatColorType.HIGHLIGHT)
+				.append(result);
+		}
+		catch (IllegalArgumentException ex)
+		{
+			builder.append(ChatColorType.HIGHLIGHT)
+				.append("Calc: Unable to calculate expression: ")
+				.append(eqn)
+				.append(" Error: Illegal arguments.");
+		}
+
```

Comment:
Remove this empty line or see previous comment

Reply:
Accidental. Fixed.